<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber_Iist">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Isoll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_Isoll">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I Soll</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I Soll</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I Ist</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>U Soll</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Usoll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_Uist">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>U Soll</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_Usoll">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>U Ist</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PWM Soll</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_PWMist">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PWM Soll</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_PWMsoll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PWM ist</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_PWMsoll">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_I">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_U">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_PWM">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
